arroyo-api/src/pipelines.rs

```diff
@@ -98,11 +108,7 @@ where
         },
     )
     .await
-    .with_context(|| "failed to generate SQL program")
-    .map_err(|err| {
-        warn!("{:?}", err);
-        bad_request(format!("{}", err.root_cause()))
-    })?;
+    .with_context(|| "failed to generate SQL program")?;
```

Comment:
this loses the detailed sql errors. 

For example, `select * from blah;` which on master tells me the actual error (blah doesn't exist) now just returns "failed to generate SQL program"

Reply:
Fixed